device-types/Cisco/C9120AXE-E.yaml

```diff
@@ -0,0 +1,22 @@
+---
+manufacturer: Cisco
+model: Catalyst 9120AXE-E
```

Comment:
Should include all variants of the 9120AXE (-A, -B, -H, -Q, -K, -T, -G, -D, -S, -I, -Z, -F, -N, -C, -R) if we are specifying the "-E".